logic/transaction_pool.js

```diff
@@ -107,9 +111,10 @@ function TransactionPool(
 // Public methods
 /**
  * Bounds input parameters to private variable modules.
- * @param {Accounts} accounts - Accounts module instance.
- * @param {Transactions} transactions - Transactions module instance.
- * @param {Loader} loader - Loader module instance.
+ *
+ * @param {Accounts} accounts - Accounts module instance
```

Comment:
Are we consistent in our use of lowercase and uppercase for `accounts`, `transactions` etc?

Reply:
No, probably not, but there are also no corresponding typedefs, yet.

Reply:
As discussed, we will open a new issue for standardizing typedefs in all files